backend/centraldeerros/src/main/java/br/com/guiabolso/centraldeerros/controller/EventsController.java

```diff
@@ -38,4 +39,24 @@
 			return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
 		}
 	}
+
+	@GetMapping("/event/{id}")
+	public ResponseEntity<Optional<Event>> getEvent(@PathVariable(value = "id")long id){
+		try {
+			return new ResponseEntity<>((Optional<Event>) eventService.get(id), HttpStatus.OK);
+		}catch (Exception e) {
+			return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+		}
+	}
+
+	@PostMapping("/event")
+	public ResponseEntity<Event> addEvent(@RequestBody Event event) {
+		try {
+
+			return new ResponseEntity<>((Event) eventService.add(event), HttpStatus.CREATED);
```

Comment:
sera que nao ficaria legal a gente passar o body na resposta? coloquei assim no account:
 @PostMapping("/account")
    public ResponseEntity<Account> newAccount(@Valid @RequestBody Account account){
        try{
            this.accountService.saveAccount(account);
            return ResponseEntity.status(HttpStatus.CREATED).body(account);
        } catch (Exception e){
            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
        }
    }

Reply:
ah esse metodo é pro evento, quando eu clonei nao tinha nenhum pra salvar ele, acabei fazendo esse, mas posso criar o controller da conta e fazer assim sim, fica legal

Reply:
belezinha